.github/workflows/build_all.yml

```diff
@@ -107,7 +107,8 @@ jobs:
         run: |
           pip install artifactory
           python ci-scripts/helm_repo_rotation.py
-
+  agw-build:
+    uses: ./.github/workflows/bazel.yml@master
```

Comment:
Tha bazel jobs seem to not be kicked off correctly:

![2023-06-15_16-12-20](https://github.com/magma/magma/assets/1187050/bf1f9a5d-9417-4069-8319-7641d5347cf8)

Reply:
@tapasmishra : If you have already understood that it is good and have seen that it works, you could give the approval here?

Reply:
@maxhbr You can check this repo where I have a simple scenario on this issue : https://github.com/tapaswl/test_workflow/tree/main

Reply:
I think force-merge will help to save duplicate works.